src/pages/first-byte-page/components/Window.css

```diff
@@ -0,0 +1,28 @@
+.fb-window {
+    border: 3px solid #250094;
+    padding: 8px;
+    background-color: #E8E0FF;
+    display: flex;
+    flex-direction: column;
+    margin: 100px auto;
+}
+  
+.fb-window-header {
+    border: 3px solid #250094;
+    color: white;
+    font-size: 30px;
+    margin-bottom: 8px;
+    text-align: left;
+    background-color: #7E65FE;
+    padding: 3px 10px;
+}
+  
+.fb-window-content {
+    border: 3px solid #250094;
+    background-color: white;
+    flex-grow: 1;
+    font-size: 18px;
+    padding: 3px 10px;
+
```

Comment:
I think doing overflow hidden for window content would be a good idea

Reply:
oh yes yes, just added it 👍